### DISCUSSION TITLE
PHPMailer not function

### DISCUSSION THREAD

**Discussion by sdezen** [PHPMailer/PHPMailer#2997] [General]
I'm trying to get PHPMailer 6.9.1 to work but I can't get stuck on line 22 "$mail = new PHPMailer();"
I also can't find this file
require 'vendor/autoload.php';

Can you help me put the files in the right location inside my "include_php" folder and give me a working example.
Thank you
[test_mail.txt](https://github.com/PHPMailer/PHPMailer/files/13730319/test_mail.txt)

**Comment by ItsJoshBrown** [comment]
I'm sure you have already gotten this solved by now, I'm not sure of your folder path's. But here is a working example of one I have. 

Here is my file Path structure my sendEmail.php is the working file for all the require '../vendor/autoload.php'. 

but it's also sort of dependent on where vendor folder is located for the relative paths. 

```
.
├── www
├── emailCode/
│   └── sendEmailTest.php
└── vendor/
    └── autoloader.php
```


```PHP
use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\SMTP;

// reference the autoloader.php file in my the vendor folder path
require '../vendor/autoload.php';

/** 
* Alternatively as long as the file is in the same directory you can do 
* require __DIR__  '/vendor/autoload.php';
*/
```
